website/content/docs/v0.15/Learn More/concepts.md

```diff
@@ -3,10 +3,146 @@ title: "Concepts"
+of operators.
+This _also_ means that the operational architecture is fundamentally different.
+
+Properly managed (which Talos Linux does), `etcd` should never have split brain
+and should never encounter noticeable down time.
+In order to do this, though, `etcd` maintains the concept of "membership" and of
+"quorum".
+In order to perform _any_ operation, read _or_ write, the database requires
+quorum to be achieved.
+That is, a _strict_ majority must agree on the current leader, and absenteeism
+counts as a negative.
+In other words, if there are three registered members (voters), at least two out
+of the three must be actively asserting that the current master _is_ the master.
+If any two disagree or even fail to answer, the `etcd` database will lock itself
+until quorum is again achieved in order to protect itself and the integrity of
+the data.
+This is fantastically important for handling distributed systems and the various
+types of contention which may arise therein.
+
+This design means, however, that having an incorrect number of members can be
+devastating.
+Having only two controlplane nodes, for instance, is mostly _worse_ than having
+only one, because if _either_ goes down, your entire database will lock.
+You would be better off just making periodic snapshots of the data and restoring
+it when necessary.
+
+Another common situation occurs when replacing controlplane nodes.
+If you have three controlplane nodes and replace one, you will not have three
+members, you will have four, and one of those will never be available again.
+Thus, if _any_ of your three remaining nodes goes down, your database will lock,
+because only two out of the four members will be available:  four nodes is
+_worse_ than three nodes!
+So it is critical that controlplane members which are replaced be removed.
+Luckily, the Talos API makes this easy.
+
+## Bootstrap once
+
+In the old days, Talos Linux had the idea of an `init` node.
+The `init` node was a "special" controlplane node which was designated as the
+founder of the cluster.
+It was the first, was guaranteed to be the elector, and was authorised to create
+a cluster...
+even if one already existed.
+This made the formation of a cluster cluster really easy, but it had a lot of
+down sides.
+Mostly, these related to rebuilding or replacing that `init` node:
+you could easily end up with a split-brain scenario in which you had two different clusters:
+a single node one and a two-node one.
+Needless to say, this was an unhappy arrangement.
+
+Fortunately, `init` nodes are gone, but that means that the critical operation
+of forming a cluster is a manual process.
+It's an _easy_ process, consisting of a single API call, but it can be a
+confusing one, until you understand what it does.
+
+Every new cluster must be bootstrapped exactly and only once.
+This means you do NOT bootstrap each node in a cluster, not even each bootstrap
```

Comment:
"not even each bootstrap node" sounds strange to me. Typo or am I just not reading it right?

Reply:
Hah, that second should read "control plane", not bootstrap.